clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -558,12 +558,14 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<ByteBuffer> fut
                 markCoordinatorUnknown();
                 log.debug("Attempt to join group failed due to obsolete coordinator information: {}", error.message());
                 future.raise(error);
+            } else if (error == Errors.FENCED_INSTANCE_ID) {
+                log.error("Received fatal exception: group.instance.id {} gets fenced", groupInstanceId);
```

Comment:
We can leave the instance id out of this message since we added it to the log context. There are a few more of these below.

Reply:
Sounds good!